projects/extension/sql/idempotent/013-loader.sql

```diff
@@ -0,0 +1,14 @@
+-------------------------------------------------------------------------------
+-- loader_file_loader
+create or replace function ai.loader_from_document
```

Comment:
naming considerations:

1. our other configs for vectorizer end in `ing`: `embedding`, `formatting`, `chunking`. Should we follow the same convention?
2. would `object_storage` be more appropriate than `document`? We should describe "where" instead of "what". We may load documents from other places in the future